tensilelite/Tensile/Tests/common/gemm/lsu_i8.yaml

```diff
@@ -0,0 +1,184 @@
+          - [16, 16, 32, 1,  1,   2, 2,  1,1 ]
+        - WorkGroup:
+          - [16,4,1]
+          - [4,4,2]
+          - [4,4,4]
+        - GlobalReadVectorWidthA: [8]
+        - GlobalReadVectorWidthB: [8]
+        - PrefetchGlobalRead: [2]
+        - PrefetchLocalRead: [1]
+        - ClusterLocalRead: [1]
+        - NumElementsPerBatchStore: [0]
+        - DepthU: [128]
+        - VectorWidthA: [1,2,-1]
+        - VectorWidthB: [1,2,-1]
+        - MIArchVgpr: [0]
+        - LocalWritePerMfma: [-1]
+        - StaggerU: [4]
+        - StaggerUStride: [256]
+        - StaggerUMapping: [2]
+        - WorkGroupMapping: [1]
+        - ScheduleIterAlg: [3]
+        - ExpandPointerSwap: [0]
+        - TransposeLDS: [0,1,2]
+        - LdsBlockSizePerPadA: [-1]
+        - LdsBlockSizePerPadB: [-1]
+        - StorePriorityOpt: [0]
+        - VectorStore: [-1]
+        - StoreSyncOpt: [0]
+        - LdsPadA: [-1]
+        - LdsPadB: [-1]
+        - 1LDSBuffer: [1]
+        - GlobalSplitU: [1,16]
+        - SourceSwap: [0, 1]
+      BenchmarkJoinParameters:
+      BenchmarkFinalParameters:
+        - ProblemSizes:
+          - Exact: [1,     127, 1, 127]
+          - Exact: [2,     127, 1, 127]
+          - Exact: [3,     127, 1, 127]
+          - Exact: [1,     1,   1, 127]
+          - Exact: [127,   1,   1, 127]
+          - Exact: [127,   2,   1, 127]
+          - Exact: [127,   3,   1, 127]
+          - Exact: [127,   127, 1, 127]
+          - Exact: [128,   128, 1, 128]
+          - Exact: [129,   129, 1, 129]
+          - Exact: [127,   127, 1, 128]
+          - Exact: [127,   127, 1, 129]
+          - Exact: [127,   128, 1, 640]
+          - Exact: [129,   128, 1, 640]
+          - Exact: [129,   128, 1, 4096]
+
+  ########################################
+  # LSU2/4 - (I8I8I)
+  ########################################
+  -
+    - # ProblemType
+      OperationType: GEMM
+      DataType: I8
+      DestDataType: I8
+      ComputeDataType: I
+      HighPrecisionAccumulate: True
+      TransposeA: 0
+      TransposeB: 0
+      UseBeta: True
+      Batched: True
+    - # BenchmarkProblemSizeGroup - Standard - All problem
+      InitialSolutionParameters:
+      BenchmarkCommonParameters:
+        - KernelLanguage: ["Assembly"]
+      ForkParameters:
+        - MatrixInstruction:
+          - [16, 16, 32, 1,  1,   1, 1,  2,1 ]
+          - [16, 16, 32, 1,  1,   2, 1,  1,2 ]
+          - [16, 16, 32, 1,  1,   1, 2,  2,1 ]
+          - [16, 16, 32, 1,  1,   1, 4,  1,2 ]
+          - [16, 16, 32, 1,  1,   2, 2,  2,1 ]
+          - [16, 16, 32, 1,  1,   4, 1,  2,1 ]
+          - [16, 16, 32, 1,  1,   1, 1,  1,1 ]
+          - [16, 16, 32, 1,  1,   2, 1,  1,1 ]
+          - [16, 16, 32, 1,  1,   4, 1,  1,1 ]
+          - [16, 16, 32, 1,  1,   1, 2,  1,1 ]
+          - [16, 16, 32, 1,  1,   1, 4,  1,1 ]
+          - [16, 16, 32, 1,  1,   2, 2,  1,1 ]
+        - WorkGroup:
+          - [16,4,1]
+          #- [8,4,2]
```

Comment:
LSU disabled?

Reply:
This was an experimental change.
I was suspecting CI error was not related to my change.
I will make some more experimental change in this PR and try I8 test without my change (to see if the error is related to my change or not).
So far, I am unable to reproduce the CI error in my local.